File: rxhttp/src/main/java/com/senierr/http/converter/DefaultConverter.java
package com.senierr.http.converter;

import io.reactivex.annotations.NonNull;
import okhttp3.Response;

/**
 * 默认转换器
 *
 * @author zhouchunjie
 * @date 2018/8/28
 */
public class DefaultConverter implements Converter<Response> {

    public @NonNull Response convertResponse(@NonNull Response response) throws Throwable {
        return response;
    }
}

File: rxhttp/src/main/java/com/senierr/http/converter/DefaultConverter.kt
package com.senierr.http.converter

import okhttp3.Response

/**
 * 默认转换器
 *
 * @author zhouchunjie
 * @date 2018/8/28
 */
class DefaultConverter : Converter<Response> {

    override fun convertResponse(response: Response): Response {
        return response
    }
}
